reactDashboard/src/components/Footer/Footer.css

```diff
@@ -66,3 +71,31 @@
 footer {
   padding-bottom: 30px;
 }
+
+.darkmode footer {
+  background-color: #121212;
+  padding-bottom: 30px;
+}
+
+.darkmode footer .sitemap .entry {
```

Comment:
Put these blocks immediately after the blocks they override please

Reply:
✅

Reply:
refactored